.github/workflows/deploy-dev.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         with:
           node-version: 18
       - run: yarn install --frozen-lock-file
-      - run: ./build.sh http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com
+      - run: ./build.sh http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com development
```

Comment:
@AndyClifford Suggestion: Only pass `mode` to the script. Let the script handle the extra complexity of setting the `--site` based on the mode